packages/flutter/lib/src/services/asset_bundle.dart

```diff
@@ -212,11 +213,15 @@ abstract class CachingAssetBundle extends AssetBundle {
 
 /// An [AssetBundle] that loads resources using platform messages.
 class PlatformAssetBundle extends CachingAssetBundle {
+
+  /// The messenger which sends the bytes for this channel, not null.
```

Comment:
not: I would probably just remove this comment. It's a private member anyways and the comment is not really telling me anything.

Reply:
Done.